src/omero_cli_render.py

```diff
@@ -134,6 +134,51 @@
     # be updated individually.
 """
 
+BATCHSET_HELP = """Set rendering settings from a hierachy of rendering
+    definitions corresponding to an OMERO container hierarchy.
+
+    'projects' and 'datasets' are supported as top level containers, 'name'
+    must be provided.
+    If multiple containers have the same name they will all be processed.
+    Top level keys beginning with '_' are ignored, so they could for example
+    be used as YAML anchors.
+
+    Definitions can be applied to projects, datasets, or datasets in projects.
+
+    'renderdef' is a rendering definition as passed to 'set', except that
+    'version' is omitted.
```

Comment:
I thought it would simplify things if we avoided mixed versioning though that's probably not a major problem to support. You're right that it should be specified somewhere though, so the question is should we require the version
- as part of every renderdef
- as part of the top-level version for this file i.e. version indicates the version of the renderdef spec + the version of this hierarchical definition
- in a dedicated top-level `renderdef_version` key
And if renderdef has it's own version indicator how do we version this  hierarchy spec?